packages/terra-paginator/src/_PaginatorButton.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { disableFocusActiveStyles, disableFocusOnBlur, enableFocusActiveStyles } from './_paginationUtils';
+import styles from './Paginator.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Sets aria-current attribute.
+   */
+  ariaCurrent: PropTypes.oneOfType([
+    PropTypes.bool,
+    PropTypes.string,
+  ]),
+  /**
+   * Sets aria-disabled attribute.
+   */
+  ariaDisabled: PropTypes.bool,
+  /**
+   * Sets aria-label attribute.
+   */
+  ariaLabel: PropTypes.string,
+  /**
+   * Child elements passed in to buttons.
+   */
+  children: PropTypes.node,
+  /**
+   * Passed in CSS className.
+   */
+  className: PropTypes.string,
+  /**
+   * Callback function triggered when clicked.
+   */
+  onClick: PropTypes.func,
+};
+
+function PaginatorButton(props) {
+  let shouldShowFocus = true;
+
+  const handleFocus = (event) => {
+    if (shouldShowFocus) event.currentTarget.setAttribute('data-focus-styles-enabled', 'true');
+  };
+
+  const handleMouseDown = () => {
+    // Prevent button from showing focus styles when clicked
+    shouldShowFocus = false;
+    // Wait until after onFocus has been triggered on browsers where it will get triggered for click
+    setTimeout(() => { shouldShowFocus = true; });
+  };
+
+  const {
+    ariaCurrent,
+    ariaDisabled,
+    ariaLabel,
+    children,
+    className,
+    onClick,
+  } = props;
+
+  return (
+    <button
+      aria-current={ariaCurrent}
+      aria-disabled={ariaDisabled}
+      aria-label={ariaLabel}
+      className={cx(className)}
```

Comment:
Should we be binding the PaginatorButton's class styles to the already bound styles passed ? Looking at some of these changes, it looks like this prop is being used like:
```
<PaginatorButton
    className={cx(['nav-link', selectedPage === totalPages ? 'is-disabled' : null])}
/>
```

Reply:
I changed the paginator button to just use the passed down props = [3c4f04a](https://github.com/cerner/terra-core/pull/2686/commits/3c4f04a82bd80d77dc4fc4f35ba36c5d85deb003)